wiki/Game_modifier/Cinema/id.md

```diff
@@ -0,0 +1,40 @@
+---
+stub: true
+tags:
+  - cinema
+  - CM
+  - mod
+  - game modifier
+---
+
+# Cinema (mod)
+
+![Ikon mod CM](/wiki/shared/mods/CM.png "Ikon mod Cinema (CM)")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*\
+*Berbeda konteks dengan [Cinematography](https://en.wikipedia.org/wiki/Cinematography "Wikipedia").*
+
+## Tentang
+
+- Singkatan: CM
+- Jenis: Spesial
+- Pengali Skor: 1.00x
+- Tombol shortcut: `V` `V` atau `Shift`+`V` (atau tahan `Ctrl`+`Shift` saat sedang memilih sebuah [beatmap](/wiki/Beatmaps))
+- Mode permainan yang cocok: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
+
+## Deskripsi
+
+*Perhatian: Penggunaan mod Auto akan menyebabkan permainan menjadi unranked.*
+
+Mod **Cinema** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan agar pemain dapat bersantai dan menonton video atau storyboard dari sebuah beatmap tanpa munculnya [hit object](/wiki/Hit_object). Mod ini sama seperti mod [Auto](/wiki/Game_modifier/Auto), tetapi **hanya background dan storyboard yang ditampilkan**.
+
+Karena mod Cinema merupakan variasi dari mod Auto, hitsound yang berasal dari hit object akan tetap terdengar. Namun elemen tayangan ulang dan layar hasil akan dinonaktifkan bersamaan dengan dim background yang disetel ke 0%.
+
+![Gameplay Auto vs. Cinema](img/GM_Cinema.jpg)
```

Comment:
Baris ini sudah berubah dari `![Gameplay Auto vs. Cinema](img/GM_Cinema.jpg)` menjadi `![Auto vs. Cinema gameplay](img/CM-comparison.jpg "Comparison of the Auto mod (left) vs the Cinema mod (right)")`. Sumber:  [[EN] Cinema (mod)](https://github.com/ppy/osu-wiki/blob/master/wiki/Game_modifier/Cinema/en.md)

Reply:
on my way